composer.json

```diff
@@ -77,6 +77,9 @@
   "conflict": {
     "drupal/drupal": "*"
   },
+  "config": {
+    "platform": {"php": "7.0.17"}
```

Comment:
This should ensure that no dependencies come down which will be incompatible with the PHP version on our production environment.

Reply:
agreed. Do you need the patch version or can it just be `7.0`?